### PR TITLE
Update Apache Http Components

### PR DESCRIPTION
**Pull request by dariusgm**
Hope this fixes various issues as described in #102

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
 		<slf4j.version>1.7.21</slf4j.version>
 		<logback.version>1.1.7</logback.version>
 		<guava.version>19.0</guava.version>
-		<apache.http.components.version>4.4</apache.http.components.version>
+		<apache.http.components.version>4.5.2</apache.http.components.version>
```
Comment (yasserg) [inline pom.xml]: 4.5.3 is the latest version

Comment (dariusgm) [inline pom.xml]: I am sorry but I can not find a version 4.5.3 in the maven repository: http://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient

Comment (yasserg) [inline pom.xml]: http://search.maven.org/#search%7Cga%7C1%7Ca%3A%22httpcomponents-client%22

Comment (dariusgm) [inline pom.xml]: They changed the artifact Id - on the configured oracle maven mirror this artifact is at the moment not available. At the moment, no further update is possible without more changes in the pom.

**Pull request merged**